test/io/wkt/wkt.cpp

```diff
@@ -46,6 +46,15 @@ void check_wkt(G const& geometry, std::string const& expected)
                       boost::to_upper_copy(expected));
 }
 
+template <typename G>
```

Comment:
I think you should also update copyright notice, adding yourself.

Reply:
What year should I mention in the copyright notice?

Reply:
Year of your contribution.

See also FAQ on maintaining copy notices here https://www.boost.org/users/license.html